api/views/course_admin.py

```diff
@@ -16,11 +18,19 @@ class CourseAdminViewSet(viewsets.GenericViewSet):
     permission_classes = [TeacherAccessPermission, ]
     queryset = CanvasCourse.objects.all()
     serializer_class = CourseSerializer
+    ordering_fields = []
+    filter_backends = [filters.OrderingFilter, DjangoFilterBackend]
 
     @action(detail=True, methods=['get'], url_path='registered-users')
     def registered_users(self, request, pk=None):
+        self.ordering_fields = ['id', 'user__username', 'user__first_name', 'status']
```

Comment:
This is redundant here, move this up to the class variable declared above

Reply:
Done.